Prims_Algorithm/Prims_Algorithm.php

```diff
@@ -0,0 +1,89 @@
+									
```

Comment:
Where is php tag?
How you ran this file?

Reply:
I just deleted whole because I haven't getting output from it so in next file I added I/O and correct code too